web-console/lib/keywords.ts

```diff
@@ -19,129 +19,120 @@
 // Hand picked from https://druid.apache.org/docs/latest/querying/sql.html
 
 export const SQL_KEYWORDS = [
-  'EXPLAIN PLAN FOR',
-  'WITH',
-  'AS',
-  'SELECT',
   'ALL',
+  'AND',
+  'ANY',
+  'AS',
+  'ASC',
+  'BETWEEN',
+  'BOTH',
+  'CASE',
+  'CENTURY',
+  'CLUSTERED BY',
+  'CROSS',
+  'CSV',
+  'CUBE',
+  'CURRENT',
+  'DAY',
+  'DAY',
+  'DAYS',
+  'DECADE',
+  'DESC',
   'DISTINCT',
+  'DOW',
+  'DOW',
```

Comment:
do dupes matter? also this doesn't work in druid, though I guess calcite recognizes it since i see it in the parser stuff :shrug: